test/standalone/test-mysql-pool.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var common = require('../hooks/common.js');
+var assert = require('assert');
+var http = require('http');
+var semver = require('semver');
+
+// hapi 13 and hapi-plugin-mysql uses const
```

Comment:
It is nicer to feature test rather than look for specific version numbers. The capability that we are really looking for is whether the version of node we are running on supports `const`.

This more of an observation at this point. If we start doing this in other places, it would be worth changing at that point.